### PR TITLE
Add disable success page config[Platform Only]

### LINKED ISSUE #5446: [CAL-340] Toggle to disable success page when success redirect is enabled
When using the event "redirect" feature to another website, the redirect shows the confirmation page still then goes to the redirect, is it possible to add a toggle to disable showing the confirmation page

— Requested by a customer

cc @shirazdole

> From [SyncLinear.com](https://synclinear.com)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-340](https://linear.app/calcom/issue/CAL-340/toggle-to-disable-success-page-when-redirect-is-enabled)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes #5446

Demo -> https://www.loom.com/share/5f24ebce4ada49c785bdd6e086ebe7da

**Environment**: Production

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/ui/components/form/checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ const CheckboxField = forwardRef<HTMLInputElement, Props>(
                     {...rest}
                     ref={ref}
                     type="checkbox"
+                    disabled={disabled}
```
Comment (hariombalhara) [inline packages/ui/components/form/checkbox/Checkbox.tsx:52]: Disabled support was missing in native checkbox

File: packages/ui/components/form/checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -59,7 +60,7 @@ const CheckboxField = forwardRef<HTMLInputElement, Props>(
                     )}
                   />
                 </div>
-                <span className="text-sm ltr:ml-3 rtl:mr-3">{description}</span>
+                <span className="text-sm">{description}</span>
```
Comment (hariombalhara) [inline packages/ui/components/form/checkbox/Checkbox.tsx:63]: Before:
Screenshot 2023-01-01 at 1 35 42 PM

After:
Screenshot 2023-01-01 at 1 36 20 PM

**Review by hariombalhara — COMMENTED** [review]
Self reviewed

File: packages/ui/components/form/checkbox/Checkbox.tsx (hunk 2 above)
Comment (JeroenReumkens) [reply packages/ui/components/form/checkbox/Checkbox.tsx:63]: Look a lot better 🙏

**Review by JeroenReumkens — APPROVED** [review]

**Comment by aasargsian** [comment]
@hariombalhara Could you pls kindly check out a related bug: https://github.com/calcom/cal.com/issues/6497

**Pull request merged**